Harvest.Web/ClientApp/src/Quotes/ActivityForm.tsx

```diff
@@ -56,8 +58,54 @@ export const ActivityForm = (props: Props) => {
     <div className="card-wrapper mb-4 no-green">
       <div className="card-content">
         <div className="row justify-content-between align-items-end">
-          <div className="col-md-8">
-            <label> Activity Name</label>
+          <div className="col-md-12">
+            <div className="row justify-content-between">
+              <div className="col-md-6">
+                <label> Activity Name</label>
+              </div>
+
+              <div className="col-md-6 text-right">
+                <button className="btn btn-link btn-sm">
+                  Adjust Year <FontAwesomeIcon icon={faCalendarWeek} />
```

Comment:
Do we still need adjust year if we have that dropdown toggle?

Reply:
I left that there as a reference to how i want it to look once the other one is hooked up